mergeable/src/GitHub.ts

```diff
@@ -0,0 +1,21 @@
+const OPEN_PRS = { state: 'open' }
+
+const fetchOpenPullRequests = (gitHubApi: GitHubApi.Instance) => async (githubSecrets: GitHubSecrets): Promise<any> => {
```

Comment:
do you know what the details pr returns? cause then you might want to type this to return `Promise<DetailType[]>`